test/codepen-projects.js

```diff
@@ -16,6 +16,7 @@
 
 // Mocha or Selenium is creating more than the max default of 10 events
 // emitters, so we increase the default max here.
+
```

Comment:
We usually don't put a blank line between inline comments and code. But if we did want to reformat this file, does it really belong in this PR?

Reply:
I think I actually included this blank line by accident. I can certainly remove it and resubmit it without the blank line.

Reply:
Yes, this file has nothing to do with the PR, so let's get rid of this change. I'll be happy to merge this PR once this change is removed.